spec/controllers/students_controller_spec.rb

```diff
@@ -541,4 +541,50 @@ def make_request(query)
     end
   end
 
+  describe '#sped_referral' do
+    let(:educator) { FactoryGirl.create(:educator, :admin, school: school) }
+    let(:school) { FactoryGirl.create(:school) }
+    let(:service) { FactoryGirl.create(:service, student: student) }
+    let(:student) { FactoryGirl.create(:student, :with_risk_level, school: school) }
+    let(:student_school_year) { FactoryGirl.create(:student_school_year, student: student) }
+
+    def make_request(options = { student_id: nil, format: :pdf })
```

Comment:
I'm not familiar with RSpec. I copy and pasted my way into this. I'm not confident that RSpec actually makes real HTTP  requests. It's probably just a simulation. To my understanding, these tests will fake a request and check the values of the instance variables passed to the template.